app/controllers/inbox/meetings.js

```diff
@@ -14,4 +16,60 @@ export default class InboxMeetingsController extends Controller {
   get readOnly() {
     return !this.currentSession.canWrite && this.currentSession.canRead;
   }
+
+  isInstallationMeeting = (meeting) => {
+    return meeting instanceof InstallatieVergaderingModel;
+  };
+
+  @action
+  async createInstallationMeeting() {
```

Comment:
This method takes a while, even locally. I think we need to add something to give feedback to the user that something is happening. Only part of the delay is down to this method, as it also looks like there's something of a waterfall of requests after it has created the meeting. Strangely it doesn't have such a strong impact when simply loading an existing meeting.

Reply:
Actually, looking more closely, it's the POST requests to create the new agendapoints and treatments that seem to be cascading. It would be nice if we can get ember-data to handle this better, but that's likely a slippery slope...

Reply:
Yeah... I tried to batch the requests as much as possible (using `Promise.all`), but do not really know if there is a better way.
Will see if I can add a loading indicator.